spec/unit/recipes/default_spec.rb

```diff
@@ -34,50 +33,13 @@
           .with(
             owner: 'root',
             group: 'root',
-            mode: 0750,
+            mode: '750',
             recursive: true
           )
       end
       it do
         expect(chef_run).to_not run_execute('ganeti-initialize')
       end
-      context 'systemd' do
-        cached(:chef_run) do
-          ChefSpec::SoloRunner.new(p).converge(described_recipe)
-        end
-        before do
-          allow(File).to receive(:exist?).and_call_original
-          allow(File).to receive(:exist?).with('/etc/init.d/ganeti').and_return(false)
-        end
-        it do
-          expect(chef_run).to enable_service('ganeti').with(
-            service_name: 'ganeti.target',
-            supports: { status: true, restart: true }
-          )
-        end
-        it do
-          expect(chef_run).to start_service('ganeti').with(
-            service_name: 'ganeti.target',
-            supports: { status: true, restart: true }
-          )
-        end
-        %w(
-          ganeti-confd
-          ganeti-kvmd
-          ganeti-luxid
-          ganeti-noded
-          ganeti-mond
-          ganeti-rapi
-          ganeti-wconfd
-        ).each do |ganeti_service|
-          it do
-            expect(chef_run).to enable_service(ganeti_service).with(supports: { status: true, restart: true })
-          end
-          it do
-            expect(chef_run).to start_service(ganeti_service).with(supports: { status: true, restart: true })
-          end
-        end
-      end
```

Comment:
These tests should remain, but the context is no longer needed I think.

Reply:
I believe these tests already exist below, the CentOS 6 cleanup resulted in removing the configuration path where `/etc/init.d/ganeti` exists. Given those checks were removed from the recipe, I just adjusted some of the other tests in the main context (specifically the ganeti service name and not starting the helper services) 
https://github.com/osuosl-cookbooks/ganeti/blob/c1785f917560add22aec6db3327954ffb4b58ad4/spec/unit/recipes/default_spec.rb#L90-L116 to reflect what should always happen now that CentOS6 is gone.